React/my-app/src/screens/home/App.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import data from "./data.json";
+import "./App.css";
+import defaultImg from "./photos/default.png";
+import { WBOOKS, SELECCIONAR_FILTRO, TITULO, AUTOR } from "./strings.js";
+
+class App extends Component {
+  state = { books: data, searchType: null };
+
+  render() {
+    return (
+      <div className="mainContainer">
```

Comment:
all this html code should be on a `layout` file at same level that this index file.
move this and then import it like `Home`

`import Home from './layout';`